Common.xml

```diff
@@ -3434,14 +3434,14 @@ Using values in range 0-127 is recommended to avoid ambiguities with byte order
         <TSLink>0</TSLink>
       </Item>
       
-        <Item ID="6042">    
-        <Name>Reserved</Name>     
-        <Operations></Operations>     
-        <Type></Type>
-        <RangeEnumeration></RangeEnumeration>
+      <Item ID="6042">    
+        <Name>Quality</Name>     
+        <Operations>R</Operations>     
+        <Type>Integer</Type>
+        <RangeEnumeration>-8999..8999</RangeEnumeration>
         <Units></Units>
-        <Submitter></Submitter>
-        <Description></Description>
+        <Submitter>Vaisala</Submitter>
+        <Description>Quality value indicates how reliable sensor measurement is. This coding is vendor specific.</Description>
```

Comment:
IPSO Group agrees to potentially merge the object with the following added to the description:
INVALID (-8500) : Measurement value shall not be used
UNCHECKED (0) : Measurement value has not undergone any quality checks
SUSPICIOUS (6500) : Measurement value contains uncertainty, for example statistic with low number of samples
ACCEPTED (8500): Measurement value is good for use